partial-update-tests/generated/src/main/java/fixtures/bodystring/implementation/EnumsImpl.java

```diff
@@ -114,6 +122,7 @@ Mono<Response<String>> getReferenced(
         Mono<Response<Void>> putReferenced(
                 @HostParam("$host") String host,
                 @BodyParam("application/json") BinaryData enumStringBody,
+                @HeaderParam("Accept") String accept,
```

Comment:
Should we add headers, query params before body?

Reply:
"Accept" is one synthesized by m4, not one in swagger.

But yes, Haoling, please verify the order is same with vanilla.

Reply:
I checked vanilla, it also put 'Accept' header parameter after body parameter.

https://github.com/Azure/autorest.java/blob/main/vanilla-tests/src/main/java/fixtures/bodystring/implementation/EnumsImpl.java#L59-L66